### PR TITLE
[yomm2 ] Bump yomm2 to 1.3.1

### PR DESCRIPTION
**Pull request by stellarpower**
Existing version 1.1.2 [had compile error](https://github.com/jll63/yomm2/issues/26) so I have nbumped to the latest upstream tag (1.3.1)

I don't have the availability to maintain this ATM, it's not properly tested, not been built cross platforms, etc. etc. I have just changed the patches from the previous version that wouldn't apply and pushed my work up so that someone else can benefit.

Cheers

- [ ] Changes comply with the [maintainer guide](https://github.com/microsoft/vcpkg-docs/blob/main/vcpkg/contributing/maintainer-guide.md)
  * I think so.
- [x] SHA512s are updated for each updated download
- [?] The "supports" clause reflects platforms that may be fixed by this new version
- [?] Any fixed [CI baseline](https://github.com/microsoft/vcpkg/blob/master/scripts/ci.baseline.txt) entries are removed from that file.
- [x] Any patches that are no longer applied are deleted from the port's directory.
- [x] The version database is fixed by rerunning `./vcpkg x-add-version --all` and committing the result.
  * This removed my "port-version" from the manifest.
- [x] Only one version is added to each modified port's versions file.

### CODE REVIEW

**Comment by stellarpower** [comment]
@microsoft-github-policy-service agree

**Review by JonLiu1993 — CHANGES_REQUESTED** [review]

**Comment by stellarpower** [comment]
Thanks for the changes!

**Comment by JonLiu1993** [comment]
@stellarpower, please run command `./vcpkg x-add-version yomm2 --overwrite-version` and commit again.

**Comment by JonLiu1993** [comment]
Ping @stellarpower for response, could you consider the review suggestion above?

**Comment by stellarpower** [comment]
Sorry, really just not having the time for development right now.

That is pushed to the branch so hopefully should be visible.

Think I should have resolved the conflicts, but IDK why it's then committed this back to my branch. The conflicts should be with the base I presume.

**Review by BillyONeal — DISMISSED** [review]
This looks good to me.

@stellarpower can you un-draft it if you think it's OK to go, or are you waiting to do something?

@JonLiu1993 It looks like your comments have been fixed, does it look good to you now?

File: versions/y-/yomm2.json [hunk 1]
```diff
@@ -1,9 +1,9 @@
 {
   "versions": [
     {
-      "git-tree": "8bd8a2b21272e70a16c55b55215a7b5f6bd295d3",
-      "version": "1.1.2",
-      "port-version": 2
```
Comment (JonLiu1993) [inline versions/y-/yomm2.json:6]: @stellarpower, Please restore the records here and add new records, so that we can go back to the historical version.

**Review by JonLiu1993 — CHANGES_REQUESTED** [review]

File: versions/y-/yomm2.json (hunk 1 above)
Comment (JonLiu1993) [reply versions/y-/yomm2.json:6]: @BillyONeal, Yes the others seem to have been fixed but there is one place that needs to be modified.

**Comment by JonLiu1993** [comment]
@stellarpower, After completing the above modification, please click `ready for review` so that we can review megre as soon as possible, Thanks.

File: versions/y-/yomm2.json (hunk 1 above)
Comment (BillyONeal) [reply versions/y-/yomm2.json:6]: Ah, yes, old versions should never be removed from the version database. I'll push a fix.

**Review by BillyONeal — APPROVED** [review]

**Comment by BillyONeal** [comment]
@stellarpower Since it sounds like you are OK with this outcome can you undraft the PR?

**Review by JonLiu1993 — APPROVED** [review]

**Comment by JonLiu1993** [comment]
@stellarpower, please click `ready for review` so that we can review megre as soon as possible, Thanks.

**Comment by BillyONeal** [comment]
Thanks for the update!

**Comment by stellarpower** [comment]
Thanks a lot for your help!!

**Pull request merged**